lib/first_ach/client.rb

```diff
@@ -12,7 +12,8 @@ def self.execute(method, payload)
         method:  method,
         url:     url,
         headers: default_headers,
-        payload: payload
+        payload: payload,
+        timeout: 600
```

Comment:
We should add this to the configuration, and leave it to default. This allows the using app to set it, and not have to change it in code. 

See https://github.com/AquilaLoans/first_ach-ruby/blob/develop/lib/first_ach/configuration.rb